docs/sources/reference/commandline/cli.md

```diff
@@ -896,14 +896,22 @@ This will create a new Bash session in the container `ubuntu_bash`.
 
 ## export
 
-    Usage: docker export CONTAINER
+    Usage: docker export [OPTIONS] CONTAINER
 
-    Export the contents of a filesystem as a tar archive to STDOUT
+    Export the contents of a filesystem to a tar archive (streamed to STDOUT by default)
 
-For example:
+      -o, --output=""    Write to a file, instead of STDOUT
+
+      Produces a tarred repository to the standard output stream.
```

Comment:
Not sure this is the right verb. You can't "produce to" something. Need to change it to something like: "Creates a tarred repository and streams it to STDOUT." or something like that.